frontend/src/metabase/dashboard/components/ClickMappings.jsx

```diff
@@ -337,7 +337,7 @@ export function isMappableColumn(column) {
 export function clickTargetObjectType(object) {
   if (!object.dataset_query) {
     return "dashboard";
-  } else if (object.dataset_query.type === "native") {
+  } else if (new Question(object).isNative()) {
```

Comment:
Not sure if that's ok to create questions without metadata. Previously it wasn't

Reply:
I agree that this is something of a footgun. I think it's okay, though, since we're not planning on leaving this code around for longer than a couple of weeks, before it'll get refactored further.